Gives more power to admins:
* Removes the "bzadmin" motto from BZAdmin clients
* Enables anonymous /say by default
* Doesn't announce /sets to anyone but admins

diff --git a/src/bzadmin/BZAdminClient.cxx b/src/bzadmin/BZAdminClient.cxx
index 2357d04..9bce29c 100644
--- a/src/bzadmin/BZAdminClient.cxx
+++ b/src/bzadmin/BZAdminClient.cxx
@@ -73,7 +73,7 @@ BZAdminClient::BZAdminClient(BZAdminUI* bzInterface)
     // won't really output anything, just gets token
     outputServerList();
   }
-  sLink.sendEnter(TankPlayer, myTeam, startupInfo.callsign, "bzadmin", startupInfo.token);
+  sLink.sendEnter(TankPlayer, myTeam, startupInfo.callsign, "", startupInfo.token);
   if (sLink.getState() != ServerLink::Okay) {
     std::cerr << "Rejected." << std::endl;
     return;
diff --git a/src/bzfs/commands.cxx b/src/bzfs/commands.cxx
index 4d24b21..2004727 100644
--- a/src/bzfs/commands.cxx
+++ b/src/bzfs/commands.cxx
@@ -1025,7 +1025,7 @@ bool SetCommand::operator() (const char    *message,
   sendMessage(ServerPlayer, t, cmdReturn.c_str());
   snprintf(message2, MessageLen, "Variable Modification Notice by %s of %s",
           playerData->player.getCallSign(), command.c_str());
-  sendMessage(ServerPlayer, AllPlayers, message2);
+  sendMessage(ServerPlayer, AdminPlayers, message2);
   return true;
 }
 
@@ -3375,6 +3375,8 @@ bool SayCommand::operator() (const char    *message,
     return true;
   }
 
+  sendMessage(ServerPlayer, AllPlayers, messageText.c_str() + messageStart );
+
   // no anonymous messages
   messageText += " (";
   messageText += playerData->player.getCallSign();

